spec/spec_helper.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../book'
+require_relative '../classroom'
+require_relative '../decorate'
+require_relative '../person_class'
+require_relative '../rental'
+require_relative '../student_class'
+require_relative '../teacher_class'
+
+describe Book do
+  before :each do
+    @book = Book.new('All girls are the same', 'Tobby')
+  end
+
+  describe '#addnew' do
+    it 'takes the two parameter' do
+      expect(@book).to be_an_instance_of Book
+    end
+  end
+
+  describe 'check value of title and author' do
+    it 'value of title' do
+      expect(@book.title).to eq('All girls are the same')
+    end
+
+    it 'value of author' do
+      expect(@book.author).to eq('Tobby')
+    end
+  end
```

Comment:
- [ ] Great job working on the book test but I think you should test `add_rental` method that exist in book.rb file as well since it is required to Add unit tests for all the methods in all the classes so please add test for this method as well 💯🔥✅

![image](https://user-images.githubusercontent.com/58553711/233382406-8f378b42-0d27-48df-b7bf-9bb5fbd04f9e.png)